docs/pages/blog/mui-x-v6.md

```diff
@@ -0,0 +1,188 @@
+---
+title: MUI X v6 is ready for production!
+description: Introducing the new major version of the advanced components.
+date: 2023-03-06T00:00:00.000Z
+authors: ['josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-v6/card.png" alt="" style="width: 100%; margin-bottom: 16px;" />
+
+[MUI X v6.0.0](https://github.com/mui/mui-x/releases/tag/v6.0.0) is finally here with many improvements, new features, customization abilities, and a more robust foundation to accommodate the changes we want to deliver next.
+
+## ✨ Features highlights ✨
+
+### Data Grid
+
+#### [ApiRef now available for all users](/x/react-data-grid/api-object/)
```

Comment:
For the top of the blog, consider if it would make sense to copy the TOCs https://nextjs.org/blog/next-13. I like how it allows developers that have 3 minutes to get 80% of the information, while people that have 10 minutes to dive deeper.